cpp/data/sql_updates/vufind.4

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE vufind.ixtheo_id_result_sets DROP FOREIGN KEY ixtheo_id_result_sets_ibfk_1;
```

Comment:
ixtheo tables will only exist on ixtheo machines, so we need some kind of IF EXISTS logic here